src/backend/distributed/planner/fast_path_router_planner.c

```diff
@@ -162,6 +162,17 @@ FastPathRouterQuery(Query *query, Node **distributionKeyValue)
 		return false;
 	}
 
+#if PG_VERSION_NUM >= PG_VERSION_15
+	if (query->commandType == CMD_MERGE)
```

Comment:
Can we get a commandType == CMD_MERGE for PG < 15?

Reply:
Well, I guess it is better not to have that just in case someone overlooks and uses CMD_MERGE in other versions?

Reply:
No, but build fails if we don't wrap this with an if statement.